onnxruntime/core/providers/snpe/snpe_runtime_options.cc

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/providers/snpe/snpe_runtime_options.h"
+#include "core/common/logging/logging.h"
+
+namespace onnxruntime {
+namespace contrib {
+namespace snpe {
+
+void SnpeRuntimeOptions::ParseOptions() {
+  static const std::string OPT_RUNTIME = "runtime";
+  static const std::string OPT_PRIORITY = "priority";
```

Comment:
move them out of this function, and define them as:
```
constexpr const char* xxx = "Xxx"
```
Then they will compile-time const.

Reply:
moved, but the name is part of API with our customer already